scripts/create-change-file.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+ 
+let fs = require('fs');
+let inquirer = require('inquirer');
+let path = require('path')
+let guid = require('guid');
+
+let packages = [];  // keeps track of package.json files that have been found
+let answers = {};
+let data = [];
+
+// Gets all the package.json files under the current executing directory
+let getPackages = (dir, done) => {
```

Comment:
When it moves to npmx we can switch; however I actually think we do want the ability to crawl for packages, especially without the presence of an npmx.json. It would be a lot more robust.